examples/capture-instance/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	v "github.com/IBM-Cloud/power-go-client/clients/instance"
+	ps "github.com/IBM-Cloud/power-go-client/ibmpisession"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+)
+
+func main() {
+
+	//session Inputs
+	token := " < IAM TOKEN > "
+	region := " < REGION > "
+	accountID := " < ACCOUNT ID > "
+	zone := " < ZONE > "
+
+	// volume inputs
+	piID := "< POWER INSTANCE ID >"
+	volumes := make([]string, 1) //List of Data volume IDs
+	volumes[0] = "< VOLUME ID >"
+
+	//capture inputs
+	captureDestination := " < Capture Destination > "
+	captureName := " < Capture Name > "
+	instance_name := " < Name of vm > "
+	CloudStorageImagePath := " < Cloud Storage Image Path > "
+	CloudStorageAccessKey := "< Cloud Storage Access key>"
+	CloudStorageSecretKey := "< Cloud Storage Secret key>"
+	CloudStorageRegion := "< Cloud Storage Region >"
+
+	session, err := ps.New(token, region, true, accountID, zone)
+	if err != nil {
+		log.Fatal(err)
+	}
+	powerClient := v.NewIBMPIInstanceClient(context.Background(), session, piID)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	body := &models.PVMInstanceCapture{
+		CaptureDestination:    &captureDestination,
+		CaptureName:           &captureName,
+		CaptureVolumeIds:      volumes,
+		CloudStorageAccessKey: CloudStorageAccessKey,
+		CloudStorageImagePath: CloudStorageImagePath,
+		CloudStorageRegion:    CloudStorageRegion,
+		CloudStorageSecretKey: CloudStorageSecretKey,
+	}
+	createRespOk, err := powerClient.CaptureInstanceToImageCatalogV2(instance_name, body)
```

Comment:
It will be good if we are able to delete the image from catalog to clean up the data at the end.